src/components/Modal.astro

```diff
@@ -21,6 +21,21 @@ if (speakerTalk) {
   const { Content } = await speakerTalk.render();
   TalkContent = Content;
 }
+
+let links = [];
+const sortOrder = ["Twitter", "Github", "Website"];
+
+if (person.data.links) {
+  for (const [key, value] of Object.entries(person.data.links)) {
+    links.push({
+      name: key[0].toUpperCase() + key.substring(1),
```

Comment:
nitpick: the official spelling for GitHub should be with capital G and H, so you can use a mapping from the `key` to the correct display name.